tests/scene/test_code_edit.h

```diff
@@ -3456,7 +3456,7 @@ TEST_CASE("[SceneTree][CodeEdit] completion") {
 		code_edit->confirm_code_completion();
 		CHECK(code_edit->get_line(0) == "test");
 
-		/* Favorize suggestion starting from the string to complete (matching start). */
+		/* Prefer suggestion starting from the string to complete (matching start). */
```

Comment:
```suggestion
		// Prefer suggestion starting from the string over complete (matching start).
```

Reply:
No here it is to but it is not well worded I'll reword it